### PR TITLE
[OpenUV] Correcting a SAT finding.

### PR DESCRIPTION
**Pull request by clinique**
Adding alert levels consistent with MeteoAlarm and Vigicrues

Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java [hunk 1]
```diff
@@ -56,6 +59,19 @@
  */
 @NonNullByDefault
 public class OpenUVReportHandler extends BaseThingHandler {
+    private static final DecimalType ALERT_GREEN = DecimalType.ZERO;
+    private static final DecimalType ALERT_YELLOW = new DecimalType(1);
+    private static final DecimalType ALERT_ORANGE = new DecimalType(2);
+    private static final DecimalType ALERT_RED = new DecimalType(3);
+    private static final DecimalType ALERT_PURPLE = new DecimalType(4);
+
+    private static final Map<State, HSBType> ALERT_COLORS = Map.ofEntries(
```
Comment (cpmeister) [inline bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java:68]: Using `Map.of` would make this a bit cleaner, unless of course the generics checker throws a hissy fit.

File: bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java [hunk 2]
```diff
@@ -195,13 +211,17 @@ private void updateChannel(ChannelUID channelUID, OpenUVResult openUVData) {
...
-                        updateState(channelUID, openUVData.getUv());
+                        updateState(channelUID, asDecimalType(openUVData.getUv()));
+                        break;
+                    case ALERT_LEVEL:
+                        updateState(channelUID, asAlertLevel(openUVData.getUv()));
                         break;
                     case UV_COLOR:
-                        updateState(channelUID, getAsHSB(openUVData.getUv().intValue()));
+                        updateState(channelUID, ALERT_COLORS.getOrDefault(asAlertLevel(openUVData.getUv()),
+                                HSBType.fromRGB(179, 179, 179)));
```
Comment (cpmeister) [inline bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java:221]: Please cache this HSBType instance in a static final field.

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java:68]: I really like your reviews, it helps me progress. Thanks

**Comment by cpmeister** [comment]
the build is failing, looks like you need to run spotless.

**Review by cpmeister — APPROVED** [review]

**Pull request merged**